app/src/AppRouter.tsx

```diff
@@ -1,4 +1,4 @@
-import HomeRouter from 'features/home/HomeRouter';
+//import ProjectsRouter from 'features/home/HomeRouter';
```

Comment:
nit: delete line

Can delete the entire `/home` folder as well as its not being used, and doesn't have anything worth keeping in it.